### PR TITLE
Created variables to keep track of search instead of parsing URLs (#1…

### PR DESCRIPTION
**Pull request by ekager**
…065)
Different implementation that instead of parsing URLs and search engine keys, keeps track of the search terms and whether the action was a search in static variables. This should work for different geoIPs.

### CODE REVIEW

**Comment by ekager** [comment]
I think we should disable the feature for now if everyone's okay with that, I am much prouder of this way to do this, and we should keep behavior consistent not let it be working for only some users. We could polish it and then release and advertise it for the next release along with multitasking!

**Comment by pocmo** [comment]
Alright. I talk to barbara and we can also discuss this in the Focus meeting tomorrow.

**Comment by pocmo** [comment]
Can you prepare a patch to disable this for this release?

**Comment by ekager** [comment]
Yes, working on it now!

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 1]
```diff
@@ -172,7 +187,7 @@ public View inflateLayout(LayoutInflater inflater, @Nullable ViewGroup container
 
         urlView = (TextView) view.findViewById(R.id.display_url);
         updateURL(getInitialUrl());
-
+
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[Checkstyle\] ERROR: Trailing whitespace

**Pull request closed without merging**